extensions/audio/build.gradle

```diff
@@ -21,9 +21,17 @@ dependencies {
     }
     api "com.github.wendykierp:JTransforms:3.1"
 
-    runtimeOnly "org.bytedeco:ffmpeg:5.0-1.5.7:macosx-x86_64"
-    runtimeOnly "org.bytedeco:ffmpeg:5.0-1.5.7:linux-x86_64"
-    runtimeOnly "org.bytedeco:ffmpeg:5.0-1.5.7:windows-x86_64"
+    String arch = System.properties["os.arch"] == "aarch64" ? "arm64" : "x86_64"
+    String os = System.properties["os.name"].toLowerCase(Locale.ROOT)
+    if (os.contains("mac")) {
```

Comment:
We should not pick dependencies based on current os.
If we publish the package on mac (our current CI), then only mac user can use this package when download from maven.

Reply:
I see, that makes sense. I modified the change here to also include the aarch64 deps rather than pick on current os